core/src/main/scala-2.12/cats/compat/Vector.scala

```diff
@@ -1,4 +1,5 @@
-package cats.compat
+package cats
+package compat
```

Comment:
This is nitpicky but I don't think we should change these, especially since there are no other changes in this file.